scout/server/blueprints/institutes/views.py

```diff
@@ -145,58 +145,21 @@ def gene_variants(institute_id):
 
     variant_type = form.data.get("variant_type")
 
-    # check if supplied gene symbols exist
```

Comment:
Good; the update part of this is duplicated in the controller now. But, there is no user feedback from there. Perhaps add flashes with warnings there, or maybe it's possible to reuse one of the regular variant routines for that even? I'm uneasy about skipping warnings altogether - its so easy to typo a gene symbol.

Reply:
Good suggestion, I can try to re-use code already present in variantS controller, that does exactly the same thing!

Reply:
Reintroduced genes checks:

![image](https://user-images.githubusercontent.com/28093618/149774833-4164e321-67f6-487c-bf22-5ad2a46feebb.png)